### PR TITLE
Add and edit output messages and UG

### LINKED ISSUE #183: [PE-D][Tester A] Grammatical error for single event in list
There is a grammatical error when there is only 1 event in the list. It should say "There is 1 event in your list" rather than "There are 1 events in your list"

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: Dri-water/ped#4

### LINKED ISSUE #184: [PE-D][Tester B] Inconsistent Usage of grammar in menu command
In the menu command, the grammar of the menu command can be clearer.
E.g:
for the add commands, perhaps it would be better to say "adds an event" instead of "add an event" for grammatical purposes. For "copy FROM_EVENT > TO_EVENT: Copies participant list from one event to another. ", "Copies" is used here, hence it would be better for all commands to have the correct grammar.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Holy-An/ped#4

### LINKED ISSUE #192: [PE-D][Tester B] List command confusion
When the list command is prompted without any items in the list, "Here are your scheduled events: "  might be unnecessary and might cause confusion. 

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Holy-An/ped#6

### LINKED ISSUE #193: [PE-D][Tester D] Success message displayed for non-existent event timing instead of 'No event found'
This bug also applies to filtering other relevant details, where the program indicates a successful filter when there is no event found. It is a minor issue, but such messages are misleading and impact the user experience.

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: ZiliaAJY/ped#4

### LINKED ISSUE #220: [PE-D][Tester B] Grammar issue in view command
A very minor bug, perhaps it could be phrased as "There is 1 participant in ilikebird!" 

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Holy-An/ped#11

### LINKED ISSUE #224: [PE-D][Tester A] Edit event documentation confusing
The documentation for editing an event is confusing and does not clarify what EVENT and EVENT_NAME are meant to be. 

From testing, EVENT seems to refer to the name of the event to edit, and EVENT_NAME is the new name to assign to the event. This part is confusing, and could perhaps be renamed to OLD_EVENT_NAME and NEW_EVENT_NAME, or something else that is more intuitive. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Dri-water/ped#11

### PR DESCRIPTION
**Pull request by LTK-1606**
Fixes #183 
Fixes #184 
Fixes #192 
Fixes #193 
Fixes #220 
Fixes #224

### CODE REVIEW

File: src/main/java/seedu/manager/command/ListCommand.java [hunk 1]
```diff
@@ -17,7 +20,14 @@ public ListCommand() {
      * Executes the ListCommand by getting a list of all events
      */
     public void execute() {
-        StringBuilder outputMessage = new StringBuilder(String.format(LIST_MESSAGE, eventList.getListSize()) + "\n");
+        StringBuilder outputMessage;
+        if (eventList.getListSize() == 0) {
+            outputMessage = new StringBuilder(String.format(EMPTY_LIST_MESSAGE, eventList.getListSize()) + "\n");
+        } else if (eventList.getListSize() == 1) {
+            outputMessage = new StringBuilder(String.format(ONE_EVENT_LIST_MESSAGE, eventList.getListSize()) + "\n");
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/command/ListCommand.java:27]: I think you can wrap this long line?

**Review by jemehgoh — COMMENTED** [review]
Mostly looks good, just one nit and it's good to merge!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -100,7 +100,9 @@ Format:
 
 Remarks:
 
-* `TIME` must be entered in the format `yyyy-mm-dd HH:mm`.
+* `TIME` must be entered in the format `yyyy-mm-dd HH:mm`. 
+  * There is no year 0000, no month 0 and no day 0 in the AD Calendar. Using such inputs will throw an error.
+  * Throws an error if the date-time inputted is from the past.
```
[the remark this answers is not in the document]
Comment (LTK-1606) [reply docs/UserGuide.md:105]: Yes, date-times from before the current date-time will throw an error, KH is adding in that feature

[the remark this answers is not in the document]
Comment (jemehgoh) [reply docs/UserGuide.md]: Ok, looks better now.

**Review by jemehgoh — APPROVED** [review]
LGTM!

**Pull request merged**